client/state/analytics/actions.js

```diff
@@ -24,7 +24,7 @@ const joinAnalytics = ( analytics, action ) =>
 export const composeAnalytics = ( ...analytics ) => ( {
 	type: ANALYTICS_MULTI_TRACK,
 	meta: {
-		analytics: analytics.map( property( 'meta.analytics' ) )
+		analytics: analytics.map( property( 'meta.analytics[0]' ) )
```

Comment:
Whatever happens here I would like to see it resolved in another PR that merges before this one. If the analytics test has failed we need to fix it and make sure that we don't tie that subsystem into this branch. In other words, I don't want to have to re-break analytics if this PR causes a bug and we have to revert.

I'll be looking into the analytics as well. Thanks for finding this @copons.